site/assets/scss/_boosted.scss

```diff
@@ -62,13 +62,20 @@ body {
 .ods-guidelines {
   background-color: var(--bs-secondary-bg);
 
-  .card a:hover::before {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    height: 4px;
-    content: "";
-    background-color: $accessible-orange;
+  .card a{
+    &::before {
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 0;
+      content: "";
+      background-color: $accessible-orange;
```

Comment:
Note: change it to `background-color: var(--bs-link-hover-color);` in _main-jd-dark-mode_ branch when rebased